test-clients/src/main/java/com/hedera/services/bdd/suites/compose/PerpetualLocalCalls.java

```diff
@@ -91,17 +81,17 @@ private HapiApiSpec localCallsForever() {
 			@Override
 			public List<HapiSpecOperation> suggestedInitializers() {
 				return List.of(
-						fileCreate("bytecode").path(PATH_TO_CHILD_STORAGE_BYTECODE),
+						fileCreate("bytecode").path(ContractResources.CHILD_STORAGE_BYTECODE_PATH),
```

Comment:
Will it be easier to read using static import of the ```ContractResources.CHILD_STORAGE_BYTECODE_PATH``` and use ```CHILD_STORAGE_BYTECODE_PATH``` here?

Reply:
No, I don't think so. Specifying the class where the resource is taken from will let the reader know where to look for it. There have been many discussions on whether using static import is a good practice or not. I just quote the guideline from Oracle/Sun here
https://docs.oracle.com/javase/8/docs/technotes/guides/language/static-import.html

Reply:
FYI, I did use static import in `OCTokenSpec`, mostly because some lines were too long already.

Reply:
Yeah, it's kind of subtle on how and when to use static import.